must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -396,19 +396,59 @@ for ns in $namespaces; do
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}" -l "${node//./}"-debug='ready' --no-headers | awk '{print $1}')":/host/var/lib/rook/openshift-storage/crash/ "${CRASH_OUTPUT_DIR}"
     }
 
+    journal_collection(){
+        printf "collecting crash core dump from node %s \n" "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" -l "${node//./}"-debug='ready' --no-headers | awk '{print $1}')":/host/var/log/journal.gz "${JOURNAL_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" -l "${node//./}"-debug='ready' --no-headers | awk '{print $1}')":/host/var/log/kernel.gz "${KERNEL_OUTPUT_DIR}"
+
+    }
+    delete_journal_log(){
+        for node in ${nodes}; do
+        printf "collecting journal logs  from node %s \n"  "${node}"
+        # { oc debug nodes/"${node}" -- bash -c "journalctl --since --root /host "2 days ago" | gzip" >> journal.gz }
+        oc debug nodes/"${node}" << CMDS
+            chroot /host
+            cd var/log
+            rm -rf journal.gz
+            rm -rf kernel.gz
+            exit
+CMDS
+    done
+    }
     # creating a counter variable for collecting PID in array
     pids=()
+    
+    for node in ${nodes}; do
+        printf "collecting journal logs  from node %s \n"  "${node}"
+        # { oc debug nodes/"${node}" -- bash -c "journalctl --since --root /host "2 days ago" | gzip" >> journal.gz }
+        oc debug nodes/"${node}" << CMDS
+            chroot /host
+            cd var/log
+            journalctl --since "2 days ago" | gzip >> journal.gz
+            journalctl -k -b | gzip >> kernel.gz
+            exit
+CMDS
+    done
+
     # Collecting ceph crash dump
     for node in ${nodes}; do
-        printf "collecting crash and volume logs from node %s \n"  "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
+        printf "collecting crash, journal and volume logs from node %s \n"  "${node}" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
         CRASH_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/crash_${node}
         VOLUME_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/volume_collection_${node}
+        JOURNAL_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/journal_${node}
+        KERNEL_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/kernel_${node}
         mkdir -p "${CRASH_OUTPUT_DIR}"
         mkdir -p "${VOLUME_OUTPUT_DIR}"
+        mkdir -p "${JOURNAL_OUTPUT_DIR}"
+        mkdir -p "${KERNEL_OUTPUT_DIR}"
         volume_collection &
         pids+=($!)
         crash_collection &
         pids+=($!)
+        journal_collection &
+        pids+=($!)
+        delete_journal_log &
```

Comment:
This would potentially delete the files that are being collected because `journal_collection` and `delete_journal_log` would run concurrently...